script.js

```diff
@@ -0,0 +1,93 @@
+const addButton = document.getElementById('addButton');
+const booksList = document.getElementById('booksList');
+const addAuthor = document.getElementById('addAuthor');
+const addTitle = document.getElementById('addTitle');
+
+let books;
+
+if (localStorage.getItem('books') === null) {
+  books = [];
+} else {
+  books = JSON.parse(localStorage.getItem('books'));
+}
+
+function displayBooks() {
+  books.forEach((book) => {
+    const li = document.createElement('li');
+    li.innerHTML = `
+    <h2>${book.title}</h2>  
+    <p>${book.author}</p>  
+    <button class="remove">remove</button>
+    <hr>`;
+    booksList.append(li);
+  });
+}
```

Comment:
- Fix this error in console: 
![Screen Shot 2021-08-18 at 19 09 36](https://user-images.githubusercontent.com/22258274/129987713-b1dbb9ab-3f25-4924-99a3-2bc11a4b31ed.png)